src/pages/docs/index.mdx

```diff
@@ -1,3 +1,7 @@
+---
+title: docs
+---
```

Comment:
well heck. linter is unhappy with this
2:1-3:4 warning First heading level should be `1` first-heading-level remark-lint